lib/Cargo.toml

```diff
@@ -6,7 +6,7 @@ license = "MIT OR Apache-2.0"
 name = "ostree-ext"
 readme = "README.md"
 repository = "https://github.com/ostreedev/ostree-rs-ext"
-version = "0.13.0"
+version = "0.13.1"
```

Comment:
So this is actually the first release of the 0.13 (semver incompat) series, so we don't need to bump the version.

I wanted to get in https://github.com/ostreedev/ostree-rs-ext/pull/589 before we officially release 0.13.0 too.